docs/Doxyfile.in

```diff
@@ -74,20 +74,31 @@ GENERATE_BUGLIST       = NO
 
 STRIP_CODE_COMMENTS    = NO
 
+SHOW_FILES             = NO
```

Comment:
@wthrowe do you find the file pages useful?  Since we aren't documenting most files, I don't find it that useful.  If someone were to start exploring the documentation by browsing the file pages, they might get the impression we don't have much documentation...

Reply:
Yea I figured it's better to save a little disk space and generation time instead of generating those file pages. I've never used them, but I can restore this if you find them useful 🤷‍♂️

Reply:
I actually thought it would be nice to replace the Doxygen-generated links to files with links to GitHub. That just needs a little Python-postprocessing of the HTML. It's not high-priority enough for me to do right now, but I thought I'd open an issue.

Reply:
I don't use them very often, so I'm not completely opposed to them going, but one thing I do use is the information about what files the documentation came from, and that seems to have gone with them.

Reply:
ah yes, that information is useful, so I would not like to see it disappear

Reply:
👍 added this info back (`SHOW_USED_FILES = YES`)